cmd/update.go

```diff
@@ -59,6 +59,9 @@ func newUpdateCmd() *updateCmd {
 					if err != nil {
 						return err
 					}
+					if cfg.Bins[bin].Pin {
```

Comment:
Print an info message here instead of returning an error is better as otherwise, this will interrupt the whole update process.